src/components/MainContentCard.tsx

```diff
@@ -1,15 +1,37 @@
 import type { PropsWithChildren, ReactNode } from 'react';
 
-type Props = PropsWithChildren<{ header?: ReactNode; subheader?: ReactNode }>;
+type TickerItemProps = {
+  label: string;
+  value: string;
+};
+
+const TickerItem = ({ label, value }: TickerItemProps) => (
+  <div className="ml-16 h-12 leading-[48px]">
+    <span className="font-medium font-serif text-[#736D6D] mr-2">{label}</span>
+    <span className="font-bold font-serif text-mineShaft">{value}</span>
+  </div>
+);
+
+type Props = PropsWithChildren<{ header?: ReactNode }>;
+
+const MainContentCard = ({ children, header }: Props) => {
```

Comment:
MainContentCard sounds like a style agnostic to the content, but this card always shows ticker items.

please restore the `subheader` prop or rename this card to describe the job it does.

Reply:
I removed `subheader` as a prop because it will always be the same.

Just changed to `MainContentWrapper` to clarify that it contains but is not comprised entirely of the main content.

Reply:
I would still expect `MainContent` anything to be about presenting some content (any content) as primary. This component specifies the brand subheading so I'd expect a name like `BrandsContentWrapper`.

not a blocker